app/CarbonFields/Boot.php

```diff
@@ -18,13 +18,11 @@ class Boot {
 	 * @action init
 	 */
 	public function hook() {
-
-		// add_filter( 'after_setup_theme', [ $this, 'after_setup_theme' ] );
-
+		add_filter( 'after_setup_theme', [ $this, 'after_setup_theme' ] );
```

Comment:
Will this work when used as a plugin only?

Reply:
true. I'll try `plugins_loaded` for now